### PR TITLE
Removed 'is' prefix from field/param names as per #987

### PR DESCRIPTION
**Pull request by dexman545**
Kept in some places where it doesn't really fit to remove them:
	World's isClient
	NativeImage's isSbtImage
	PageTurnWidget's isNextPageButton

I believe that's all of them.
In reference to #987

### CODE REVIEW

File: mappings/net/minecraft/block/AbstractBlock.mapping [hunk 1]
```diff
@@ -534,7 +534,7 @@ CLASS net/minecraft/class_4970 net/minecraft/block/AbstractBlock
 			FIELD field_17651 exceedsCube Z
 			FIELD field_19360 collisionShape Lnet/minecraft/class_265;
 			FIELD field_19429 solidFullSquare [Z
-			FIELD field_20337 isFullCube Z
+			FIELD field_20337 FullCube Z
```
Comment (i509VCB) [inline mappings/net/minecraft/block/AbstractBlock.mapping]:
```suggestion
			FIELD field_20337 fullCube Z
```

File: mappings/net/minecraft/client/MinecraftClient.mapping [hunk 2]
```diff
@@ -7,7 +7,7 @@ CLASS net/minecraft/class_310 net/minecraft/client/MinecraftClient
 	FIELD field_1689 framebuffer Lnet/minecraft/class_276;
 	FIELD field_1690 options Lnet/minecraft/class_315;
 	FIELD field_1692 targetedEntity Lnet/minecraft/class_1297;
-	FIELD field_1693 is64Bit Z
+	FIELD field_1693 64Bit Z
```
Comment (i509VCB) [inline mappings/net/minecraft/client/MinecraftClient.mapping]: Is that a valid field name?

File: mappings/net/minecraft/client/MinecraftClient.mapping [hunk 3]
```diff
@@ -17,7 +17,7 @@ CLASS net/minecraft/class_310 net/minecraft/client/MinecraftClient
 	FIELD field_1700 instance Lnet/minecraft/class_310;
 	FIELD field_1701 openProfilerSection Ljava/lang/String;
 	FIELD field_1702 toastManager Lnet/minecraft/class_374;
-	FIELD field_1703 IS_SYSTEM_MAC Z
+	FIELD field_1703 SYSTEM_MAC Z
```
Comment (i509VCB) [inline mappings/net/minecraft/client/MinecraftClient.mapping]: Sounds kinda odd. Is there a better way to word this?

File: mappings/net/minecraft/entity/passive/HorseBaseEntity.mapping [hunk 4]
```diff
@@ -1,7 +1,7 @@
 CLASS net/minecraft/class_1496 net/minecraft/entity/passive/HorseBaseEntity
 	FIELD field_18118 PARENT_HORSE_PREDICATE Lnet/minecraft/class_4051;
 	FIELD field_6955 temper I
-	FIELD field_6956 IS_BRED_HORSE Ljava/util/function/Predicate;
+	FIELD field_6956 BRED_HORSE Ljava/util/function/Predicate;
```
Comment (i509VCB) [inline mappings/net/minecraft/entity/passive/HorseBaseEntity.mapping]: Hmm, what does this predicate do?

File: mappings/net/minecraft/world/gen/feature/ShipwreckFeatureConfig.mapping [hunk 5]
```diff
@@ -1,4 +1,4 @@
 CLASS net/minecraft/class_3172 net/minecraft/world/gen/feature/ShipwreckFeatureConfig
-	FIELD field_13803 isBeached Z
+	FIELD field_13803 neached Z
```
Comment (i509VCB) [inline mappings/net/minecraft/world/gen/feature/ShipwreckFeatureConfig.mapping]:
```suggestion
	FIELD field_13803 beached Z
```

**Review by i509VCB — CHANGES_REQUESTED** [review]
I personally think the `isClient` field on the client should become `client`, but that deserves it's own consideration in a different pr.

File: mappings/net/minecraft/client/MinecraftClient.mapping (hunk 2 above)
Comment (i509VCB) [reply mappings/net/minecraft/client/MinecraftClient.mapping]: Not valid, failed to build:
`Caused by: java.lang.IllegalArgumentException: not a valid name: 64Bit`

File: mappings/net/minecraft/client/MinecraftClient.mapping (hunk 3 above)
Comment (dexman545) [reply mappings/net/minecraft/client/MinecraftClient.mapping]: CURRENT_SYSTEM_MAC?

Comment (i509VCB) [reply mappings/net/minecraft/client/MinecraftClient.mapping]: yea that sounds better

File: mappings/net/minecraft/entity/passive/HorseBaseEntity.mapping (hunk 4 above)
Comment (dexman545) [reply mappings/net/minecraft/entity/passive/HorseBaseEntity.mapping]: Used to set the parent predicate, so that child stay around their parent

**Comment by YanisBft** [comment]
The issues with this PR are:
- Non-adjective fields are confusing without "is" prefix
- Some fields use the "is" prefix because the related json strings use it (damage source and entity predicates)

**Comment by dexman545** [comment]
Restored some old ones

File: mappings/net/minecraft/client/MinecraftClient.mapping (hunk 2 above)
Comment (liach) [reply mappings/net/minecraft/client/MinecraftClient.mapping]: Maybe `x64`?

**Review by Earthcomputer — CHANGES_REQUESTED** [review]
I'm against some of these changes in cases where the removal of the `is` prefix causes the field name to be a noun phrase rather than an adjective phrase, which seems to be quite a few of them. Some examples:
- `RealmsWorldSlotButton.State.isCurrentlyActiveSlot` -> `currentlyActiveSlot`: makes it sound like the state "has" a currently active slot, rather than "is" one.
- `AbstractBlock.isAir` -> `air`: makes it sound like an `AbstractBlock` somehow "has" air, rather than "is" air.
- You get the pattern.

Some renames that are okay because they turn them into adjectives without losing clarity:
- `BipedEntityModel.isSneaking` -> `sneaking`
- ~~`PlayerEntity.isSubmergedInWater` -> `submergedInWater`~~ (edit: apparently this one is bad because there is already `protected Entity.submergedInWater`.)
- etc

File: mappings/net/minecraft/client/Mouse.mapping [hunk 6]
```diff
@@ -3,7 +3,7 @@ CLASS net/minecraft/class_312 net/minecraft/client/Mouse
 	FIELD field_1780 activeButton I
 	FIELD field_1781 controlLeftTicks I
 	FIELD field_1782 cursorYSmoother Lnet/minecraft/class_3540;
-	FIELD field_1783 cursorLocked Z
+	FIELD field_1783 isCursorLocked Z
```
Comment (liach) [inline mappings/net/minecraft/client/Mouse.mapping]: No
```suggestion
	FIELD field_1783 cursorLocked Z
```

**Comment by liach** [comment]
@dexman545 Please remove `Closes #987` from pr description. This one **does not** fix that issue as it misses many usages.

**Comment by liach** [comment]
Mind fix the merge conflicts?

**Comment by haykam821** [comment]
Fixes #987

**Comment by YanisBft** [comment]
This PR is pretty much cursed, and has been badly merged with base branch. If you want to do it again, please make sure to take feedback here into account

**Pull request closed without merging**